kumascript/macros/XULAttr.ejs

```diff
@@ -34,12 +34,4 @@ var destOldest = lang + '/docs/XUL:' + attrName + ':' + $0;
 var destOld = lang + '/docs/XUL/' + attrName + '/' + $0;
 var destNew = lang + "/docs/" + newPath + "/" + $0;
 
-if (wiki.pageExists(destNew)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code><%
-} else if (wiki.pageExists(destOld)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destOld), dispApi))%></code><%
-} else if (wiki.pageExists(destOldest)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destOldest), dispApi))%></code><%
-} else {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code><%
-}%>
+%><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code>
```

Comment:
What it used to do on (old) line 38 is different from your code on (new) line 37. Can you take a look?

Reply:
Yeah, I'll take a look.

Reply:
Old line 38: `%><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code>`

My (new) line 37: `%><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code>`

Both are the same lines of code, let me know if I'm missing something. It is the `xxx` from 

```
if(true) {
  xxx
} else if(true) {
  yy
}

Reply:
Look at the last two characters. You didn't copy the trailing `<%`

Reply:
Ditto from below in `ifmethod.ejs`. I'll add the trailing `<%` back